wikipedia-sentences/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jina[devel,torch]==0.8.2
```

Comment:
You should also probably bump this up (together with transformers below)

Reply:
That's on my todo list. Most examples don't use latest version right now, and this will need some help from engineering team to kick it into shape for 1.0

Reply:
Now using latest ver